gradle/build-complete.gradle

```diff
@@ -0,0 +1,26 @@
+String buildNumber = System.getenv('BUILD_NUMBER')
```

Comment:
What's the reason for doing this in an included .gradle file instead of in a plugin in buildSrc? I'm not opposed to having this separation when appropriate, but I want to make sure we all agree on where the line is.

Reply:
IMO the line is when it would otherwise generally be some code that's simply in the root `build.gradle` but we want to separate it to keep that file manageable. There's not much benefit in putting this in buildSrc. Writing unit tests for this is probably overkill, and it'll only ever be applied to the root project, and won't be reused by anyone else using `build-tools`.